packages/vitest/src/types/rpc.ts

```diff
@@ -46,10 +45,13 @@ export interface ResolvedTestEnvironment {
 }
 
 export interface ContextRPC {
+  pool: Pool
+  runner: string
```

Comment:
At first I thought `runner` was related to `@vitest/runner`. Maybe `worker` instead?

Reply:
This is an interesting question. jest has runner and testRunner. What we call `runner` in jest is called `testRunner` 👀 

In this PR I also tried to standardize naming (so, everything is a worker, just not a threads worker). I think here it would be fine to name in a `worker`